src/main/java/commands/DeleteCommand.java

```diff
@@ -0,0 +1,51 @@
+package commands;
+
+import duke.Storage;
+import duke.Ui;
+import exceptions.InvalidTaskNumberException;
+import exceptions.TaskNotFoundException;
+import tasks.TaskList;
+
+/**
+ * Represents an instruction from the user to delete a task from the list.
+ */
+
+public class DeleteCommand extends Command {
+    private int index;
+
+    public DeleteCommand(int index) {
+        this.index = index;
+    }
+
+    /**
+     * Executes the command to delete the task from the TaskList.
+     * @param tasks The current TaskList.
+     * @param ui The Ui object in use.
+     * @param storage The Storage object in use.
+     * @throws TaskNotFoundException If input task number is not found in the list.
+     * @throws InvalidTaskNumberException If the user enters a non-integer argument.
+     */
+    @Override
+    public void exec(TaskList tasks, Ui ui, Storage storage) throws TaskNotFoundException,
+            InvalidTaskNumberException {
+        // ui.printDelTask(tasks, index);
```

Comment:
Once the code lines are not required, they can be removed. Else if its meant to be kept, you can consider adding TODO line so others know there are pending changes.